pom.xml

```diff
@@ -437,6 +437,11 @@
       <artifactId>tabular-client-runtime</artifactId>
       <version>1.18.4</version>
     </dependency>
+    <dependency>
```

Comment:
will attempt removing this

Reply:
definitely need this!

Reply:
just tested without it and it did not work